app_flutter/lib/logic/qualified_task.dart

```diff
@@ -25,14 +25,16 @@ const String _luciUrl = 'https://ci.chromium.org/p/flutter';
 
 @immutable
 class QualifiedTask {
-  const QualifiedTask(this.stage, this.task);
+  const QualifiedTask(this.stage, this.task, this.builder);
```

Comment:
`const QualifiedTask({this.stage, this.task, this.builder})`

This would improve the test code's readability

Reply:
Done.